server/server.js

```diff
@@ -6,21 +6,30 @@ const log = require('ololog')
 const app = express()
 
 app.use(bodyParser.json())
-
 app.use(express.static(`${__dirname}/../client/dist`))
 
-app.get('/', function (req, res) {
+app.get('/*', function(req, res) {
```

Comment:
This line will catch all requests and cause a redirect.  Should not have *

Reply:
awesome catch thank you Andy